hosts/ficolobuild/default.nix

```diff
@@ -0,0 +1,48 @@
+# SPDX-FileCopyrightText: 2023 Technology Innovation Institute (TII)
+#
+# SPDX-License-Identifier: Apache-2.0
+{
+  self,
+  config,
+  inputs,
+  lib,
+  modulesPath,
+  ...
+}: {
+  imports = lib.flatten [
+    (modulesPath + "/installer/scan/not-detected.nix")
+    inputs.disko.nixosModules.disko
+    (with self.nixosModules; [
+      common
+      user-cazfi
+      user-hrosten
+      user-jrautiola
+      user-mkaapu
+    ])
+    ./disk-config.nix
+  ];
+
+  boot.initrd.availableKernelModules = ["ahci" "xhci_pci" "megaraid_sas" "nvme" "usbhid" "sd_mod"];
+  boot.initrd.kernelModules = [];
+  boot.kernelModules = ["kvm-intel"];
+  boot.extraModulePackages = [];
+
+  # Enables DHCP on each ethernet and wireless interface. In case of scripted networking
+  # (the default) this is the recommended approach. When using systemd-networkd it's
+  # still possible to use this option, but it's recommended to use it in conjunction
+  # with explicit per-interface declarations with `networking.interfaces.<interface>.useDHCP`.
+  networking.useDHCP = lib.mkDefault true;
+
+  nixpkgs.hostPlatform = lib.mkDefault "x86_64-linux";
+  hardware.cpu.intel.updateMicrocode = lib.mkDefault config.hardware.enableRedistributableFirmware;
+
+  # Use the systemd-boot EFI boot loader.
+  boot.loader.systemd-boot.enable = true;
+  boot.loader.efi.canTouchEfiVariables = true;
+
+  # networking.hostName = "nixos"; # Define your hostname.
+  networking.networkmanager.enable = true; # Easiest to use and most distros use this by default.
+
+  # Enable the OpenSSH daemon.
+  services.openssh.enable = true;
```

Comment:
Please [import](https://github.com/tiiuae/ghaf-infra/blob/0007cdd02fb8930b1c3a5213968e24d5e436dcd2/hosts/binarycache/default.nix#L23) the nixosModule [service-openssh](https://github.com/tiiuae/ghaf-infra/blob/main/services/openssh/default.nix) instead, unless there's a good reason to enable ssh separately here.

Reply:
I will remove this line and import the service-openssh instead.